src/components/CardContainer/CardContainer.js

```diff
@@ -45,7 +61,11 @@ class CardContainer extends React.Component {
                             key={dog.id}
                             name={dog.name}
                             height={dog.height.metric + "cm"}
-                            temperament={dog.temperament}/>
+                            temperament={dog.temperament}
+                            id={dog._id}
+                            clickEvent={(id)=>{
```

Comment:
Also `clickEvent` is quite a vague name, perhaps we can call it something more relevant to the thing it's doing?

Reply:
On it